
int8 other_robot_id
int16 distance
int8 certainty